### ISSUE TITLE
[discussion] QTable: total row support

### ISSUE THREAD

**Issue by 1001v** [quasarframework/quasar#6545]
**Is your feature request related to a problem? Please describe.**
Not quite. 

**Describe the solution you'd like**
I would like to have a total row at the table, which is fixed at the bottom of the table and always visible. Maybe a new slot for tfoot will do just fine. There're some possibilities to achieve this in quasar, but none of them solves the problem. Please read below.

**Describe alternatives you've considered**
1) The first and the most obvious solution is to use position sticky at the last row, but this solution doesn't work with following cases:
- table with sorting - we always need to keep this total row at the end of the array during resort using custom sort, it's pretty hard to achieve
- virtual scroll - the bottom row doesn't present in the DOM until you scroll to the bottom, so you can't see it
2) bottom-row slot: doesn't work with virtual scroll
3) Native html tfoot: i didn't find the way to pass it to the table so it would be proper table markup.
4) A fixed block over the table: that's pretty bad solution, because it's hard to follow columns width

**Comment by hawkeye64** [comment]
I am closing the issue for inactivity. If this is still a concern, please create a conversation in the Discussions area until a solution is settled.
